src/Microsoft.Azure.WebJobs.Logging/Internal/TableScheme.cs

```diff
@@ -15,12 +15,36 @@ internal static class TableScheme
     {
         // List all partition keys in once place to ensure they're disjoint. 
         internal const string InstancePK = "I"; // InstanceTableEntity
-        internal const string TimelineAggregatePK = "T"; // TimelineAggregateEntity
-        internal const string RecentFuncIndexPK = "R"; // RecentPerFuncEntity
+        internal const string TimelineAggregatePK = "T2"; // TimelineAggregateEntity
+        internal const string RecentFuncIndexPK = "R2"; // RecentPerFuncEntity
         internal const string ContainerActivePK = "C"; // ContainerActiveEntity
-        internal const string FuncDefIndexPK = "FD"; // FunctionDefinitionEntity
+        internal const string FuncDefIndexPK = "FD2"; // FunctionDefinitionEntity x
```

Comment:
Rename because the schema is breaking. This gaurantees we won't read old schemas. 
I removed the x. 


---
In reply to: [84856799](https://github.com/Azure/azure-webjobs-sdk/pull/884#discussion_r84856799) [](ancestors = 84856799)